Makefile

```diff
@@ -36,7 +36,7 @@ endif
 
 INSTALL := /usr/bin/install
 
-MAIN_TARGET = sonic-mgmt-framework_1.0-01_amd64.deb
+MAIN_TARGET = sonic-mgmt-framework_1.0-01_$(CONFIGURED_ARCH).deb
```

Comment:
Initialize as below to initialize CONFIGURED_ARCH if incase it was not declared
CONFIGURED_ARCH ?= amd64

Reply:
done